src/coreos-assembler

```diff
@@ -43,7 +43,7 @@ cmd=${1:-}
 # commands we'd expect to use in the local dev path
 build_commands="init fetch build run prune clean list"
 # commands more likely to be used in a prod pipeline only
-advanced_build_commands="buildprep buildupload oc-adm-release upload-oscontainer"
+advanced_build_commands="buildprep encrypt buildupload oc-adm-release upload-oscontainer"
```

Comment:
Unless the command is going to gain awareness of the builds, I think the cleaner course is to leave it out of the `cosa cli` and rename it to `pkcs7encrypt` (or something like that).

Reply:
I agree that it doesn't need to be part of the `cosa cli`. Just a clarification on the rename: would you invoke the command as so `pkcs7encrypt arg1 ...` (without the cosa prefix)?

Reply:
Correct